backend/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "backend",
+  "private": true,
+  "type": "module",
+  "engines": {
+    "node": "18.x.x",
+    "npm": "9.x.x"
+  },
+  "imports": {
+    "#*": "./src/*"
```

Comment:
really bad choice. hash is a valid url parameter

import is a url

https://developer.mozilla.org/en-US/docs/Web/API/URL/hash

Reply:
https://betterprogramming.pub/the-native-way-to-configure-path-aliases-in-frontend-projects-5db70f19a6e0

Reply:
https://abhijithota.me/posts/node-import-aliases/

Reply:
these are random articles from the Internet 😄
Node has added `node:` prefixes for the imports so that the imports become valid URLs. By adding the hash we kill what they added

Reply:
I would say that we need to avoid any characters that can be used in the URL, since the Node tries to do everything so that the path in the import will be a valid URL. We make it worse

Reply:
it's not an experimental technology

Reply:
> If they try, then why did they add this feature?
it was added before the `node:` protocol. Everyone can make mistakes 🥲

Reply:
not sure if this was a mistake

Reply:
they want to make the import path a valid URL. Of course, the hash in the import is a mistake 😄

Reply:
how it defers from other import aliases?

Reply:
I would prefer to leave them because:
- they work faster
- work without build-in node environment
- no need for additional libraries
- imports autocomplete much better (the only thing is that suggestions don't work, but this easily can be fixed with ts paths, until the issue is resolved)